spec/controllers/admin/organizations_controller_spec.rb

```diff
@@ -4,6 +4,7 @@
   let!(:admin) { create(:admin) }
   let!(:organization) { create(:organization) }
   let!(:user) { create(:user) }
+  let!(:user1) { create(:user, email: 'user1@osem.io') }
```

Comment:
@shlok007 here I'm just asking a question, so instead of updating the PR with changes, let's please discuss the question, so I can understand your problems or thought process. Then, instead of applying changes on top of changes until I have no more questions left, we'll be able to just apply the correct changes once 😃